index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```

Comment:
Poner un parrafo extra
```suggestion
</table><p>este es un ejemplo</p>
```